### ISSUE TITLE
Enhance `aria` prop for `Link`and `Link Pure` to give ARIA support when opening dialog/modal components

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3211]
Scope
---
The `Link`and `Link Pure` components don't support `aria-haspopup` property for the `aria` prop. But opening a dialog with a native link is also a probable way. Therefor we have to add support for this.

Out of Scope
---
- TBD: What about other components with links (e.g. `Link Tile`)?

Notes  
---
- ICC makes heavy use of `link-pure` which open `Modal` and `Flyout` components.
- Should be equal to `Button` and `Button Pure` components.

Acceptance criteria
---
- [x] `aria-haspopup` property with all given values is supported by `Link` and `Link Pure` components.

Subtasks
---
- [ ] Task
